common/output-model.h

```diff
@@ -194,6 +168,8 @@ namespace rs2
 
         std::vector<std::shared_ptr<stream_dashboard>> dashboards;
         std::map<std::string, std::function<std::shared_ptr<stream_dashboard>(std::string)>> available_dashboards;
+        const std::vector< std::string > dashboard_names = { "Acceleration", "Frame Drops per Second", "Gyro" };
```

Comment:
Frame drops dashboard should be the first and default for new installations.

Reply:
It implemented [here](https://github.com/IntelRealSense/librealsense/pull/12671/files#diff-856cc51cbd4ce841d9d6d17652590359aa1fd207cf4bba6c352f45386c6647aeR75):

Reply:
Yes but it is second in this list.
You can also see in the viewer:
![image](https://github.com/IntelRealSense/librealsense/assets/100071798/ecd80d13-f8b1-445d-a1e2-1b202fdb8348)

Reply:
You are right. I answered according to the default dashboard only. I will change it.